types/adapters.d.ts

```diff
@@ -1,6 +1,10 @@
 export type TimeUnit = 'millisecond' | 'second' | 'minute' | 'hour' | 'day' | 'week' | 'month' | 'quarter' | 'year';
 
-export interface DateAdapterBase {
+export interface DateAdapter {
+	// Override one or multiple of the methods to adjust to the logic of the current date library.
+	override(members: Partial<DateAdapter>): void;
```

Comment:
I think we might need the definition of `new(options: unknown): void` here since we do init it that way https://github.com/chartjs/Chart.js/blob/master/src/scales/scale.time.js#L227

Reply:
If i try to add that into the interface I get an error from my IDE that interfaces cannot be constructed, only classes. So maby thats why there was a const that got exported where it was in.

Reply:
ah, ok. Good to know.